images.CI/macos/anka/Anka.Helpers.psm1

```diff
@@ -22,7 +22,9 @@ function Push-AnkaTemplateToRegistry {
         Invoke-WebRequest -Uri $uri -Method Delete | Out-Null
     }
 
-    $command = "anka registry --registry-path $RegistryUrl push --force --tag $TagName $TemplateName"
+    $AnkaConfigPath="$HOME/.config/anka/certs"
+    $AnkaCaCrtPath="$AnkaConfigPath/anka-ca-crt.pem"
```

Comment:
`AnkaConfigPath` is only used here. Could we just combine it all into `AnkaCaCrtPath`?

Reply:
Sure I dont have a preference, updating it